BLOG-POST-TEMPLATE.md

```diff
@@ -0,0 +1,202 @@
+I would put it in:
+
+`/content/2023/10`
+
+The file should be named as follows:
+
+`<Desired URL Path>.md`
+
+For example, if this was a real blog post, it would have the following path:
+
+`https://blog.maxmind.com/2023/10/BLOG-POST-TEMPLATE`
+
+Generally, you should choose a file name that contains only lowercase letters,
+numbers, and hyphens.
+
+For SEO purposes, it's good for this file name to be the same as the title of
+the blog post.
+
+Once you have the file created, you can paste the contents of this template into
+the new file and begin to modify it.
+
+## Blog Head
+
+The head of each blog post is at the very top of the file. A line with three
+hyphens opens and closes the header, as you can see above.
+
+Some attributes take an unordered list, which is part of [the basic markdown syntax](#markdown).
+
+The head contains the following attributes:
+
+| Attribute  | Description                                                                                                                                                                                                                                         |
+| ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `title`    | The title of the blog post, contained in double quotes (`""`). This will be used both as the title displayed at the top of the post in a level 1 header, and as the `<title><\title>` meta tag. [Learn more about metadata on the blog.](#metadata) |
+| `date`     | The date that the blog post is published in `YYY-MM-DD` format. Should be contained in double quotes (`""`).                                                                                                                                        |
+| `images`   | The unordered list of paths to the featured image(s) of the post. The first image is displayed on the blog home, and all images are used for social sharing. [Learn more about images below.](#images)                                              |
+| `category` | The list of categories for the post, each contained in double quotes (`""`). You can use any strings you want, so you need to manually ensure that your categories are spelled the way you'd like and are consistent with other posts.              |
+| `tag`      | The list of tags for the post, each contained in double quotes (`""`). You can use any strings you want, so you need to manually ensure that your tags are spelled the way you'd like and are consistent with other posts.                          |
+| `authors`  | The list of authors for the post, each contained in double quotes (`""`). Only the first author is displayed.                                                                                                                                       |
+
+## Metadata
```

Comment:
I suggest moving this section down, even to the last spot. The content before and after this section is about how to write a blog post, whereas this section talks about some internal, auto-generated things that the reader may not find useful while creating a blog post.